extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java

```diff
@@ -235,4 +236,15 @@ public static String join(final CharSequence delimiter, final Iterable<? extends
         }
         return stringBuilder.toString();
     }
+
+    public static String join(final String delimiter, final String mapJoin,
```

Comment:
Maybe you could add a test for this to [UtilsTest.testJoin()](https://github.com/TeamNewPipe/NewPipeExtractor/blob/dev/extractor/src/test/java/org/schabi/newpipe/extractor/utils/UtilsTest.java). But you can leave this to another PR, since the code looks correct ;-)